scripts/ci/in_container/kubernetes/setup_kubernetes_and_deploy_app.sh

```diff
@@ -14,12 +15,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-ARG AIRFLOW_CI_IMAGE
-FROM ${AIRFLOW_CI_IMAGE}
 
-COPY airflow-test-env-init.sh /tmp/airflow-test-env-init.sh
+set -euo pipefail
+MY_DIR=$(cd "$(dirname "$0")" && pwd)
 
-COPY bootstrap.sh /bootstrap.sh
-RUN chmod +x /bootstrap.sh
-
-ENTRYPOINT ["/bootstrap.sh"]
+echo
+echo "Set up Kubernetes cluster for tests"
```

Comment:
Are you sure you need this file? This file is used in only one place and does not contain complicated logic. Can you make it inline? This will reduce the number of files and make it easier to understand.

Reply:
Good point! Fixed.